Flagsmith.Client.Test/PollingManagerTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using Moq;
+using System.Threading.Tasks;
+
+namespace Flagsmith.FlagsmithClientTest
+{
+    public class PollingManagerTest
```

Comment:
This isn't really directly testing the `PollingManager` is it? It's more testing the `FlagsmithClient` and _indirectly_ testing the `PollingManager`. Since the `PollingManager` class simply expects a callback function, is there not a way that we can pass some kind of mock function in to test it in isolation?

Reply:
Now i have updated the code to directly test the `pollingManager`.